content/04.results.md

```diff
@@ -169,6 +170,26 @@ Our filtering criteria were intentionally stringent, so the increased estimate o
 Many of these missed annotations were for preprints posted in the 2017-2018 interval.
 As opposed to those published in 2019 and later, these preprints are old enough that they are likely to have been published but it was interesting that the rate was not observed to be higher for older preprints.
 
-### Journal Recommendations/Audience Associations
-1. Title will change once analysis is finished 
-2. Provide key figure for this section and take-home message
+### Recommending Journals Based on Preprint Representation
+
+![
+Both classifiers outperform the randomized baseline when predicting a paper's journal endpoint.
+This bargraph shows each model's accuracy in respect to predicting the training and test set.

```

Comment:
How is the training set accuracy in this classifier not 100%? I thought our evaluation was whether or not the eventually published journal was in the top X (where X is 10 in this case b/c of the webserver).

Reply:
>How is the training set accuracy in this classifier not 100%? I thought our evaluation was whether or not the eventually published journal was in the top X (where X is 10 in this case b/c of the webserver).

You are correct in that the evaluation is determining if the published journal appears within the top 10 nearest neighbors. The problem here is that a paper could align closely to other papers from drastically different journal endpoints rather than a paper from the true journal endpoint. These "other" papers drown out the desired signal, which causes a decrease in performance for the paper-paper classifier. Regarding the journal centroid classifier, it could be the similar issue as above. Outside journals could be publishing more papers that align better to the query rather than the true journal itself. 

The numbers in this chart are for the subsampled training set version (randomly selecting 100 random papers from each journal); however, even when using the full dataset the max performance was 62% (mega-journal influence).

Reply:
Is a paper not necessarily most similar to itself though? That's the bit that I'm confused by.

Reply:
>Is a paper not necessarily most similar to itself though? That's the bit that I'm confused by.

A paper should be most similar to itself, but might not be most similar to _other_ papers from the same journal.

Reply:
If it's most similar to itself, wouldn't its journal always be in the top 10 though?

Reply:
As discussed in person. The confusion here is the figure says evaluated on training set. This is true; however the training set was evaluated via _cross validation_. Will need to update the figure to say cross validation instead of training set to avoid any further confusion.